client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -176,6 +176,16 @@ const LoggedInForm = React.createClass( {
 		return text;
 	},
 
+	isCalypsoStartedConnection() {
+		const site = this.props.jetpackConnectAuthorize.queryObject.site.replace( /.*?:\/\//g, '' );
+		if ( this.props.jetpackConnectSessions[ site ] ) {
```

Comment:
i think this needs to be `if ( this.props.jetpackConnectSessions && this.props.jetpackConnectSessions[ site ] )`